lib/node_runner.go

```diff
@@ -63,9 +65,19 @@ func NewNodeRunner(
 	return nr
 }
 
+type HttpRe struct{}
+
+func (r HttpRe) ResponseMessage(w http.ResponseWriter, o string) {
+	fmt.Fprintf(w, string(o))
+}
+
+func (r HttpRe) ReceiveMessage(t *sebaknetwork.HTTP2Network, msg sebaknetwork.Message) {
+	t.ReceiveChannel() <- msg
+}
+
 func (nr *NodeRunner) Ready() {
 	nr.network.SetContext(nr.ctx)
-	nr.network.Ready()
+	nr.network.Ready(HttpRe{})
```

Comment:
To define `HttpRe` in here does not seem good idea. `NodeRunner` was designed to be neutral from network types. I suggest instead of set `HttpRe` in here, how about creating new method, `SetRe(Re)` in `sebaknetwork.Network` and `sebaknetwork.Network.Ready()` can use it when `Ready()`.

Reply:
I wholeheartedly agree with you 👍